applications/asset_tracker/src/main.c

```diff
@@ -1750,19 +1753,33 @@ void handle_bsdlib_init_ret(void)
 #endif /* CONFIG_BSD_LIBRARY */
 }
 
+static void no_sim_go_offline(struct k_work *work)
+{
+#if defined(CONFIG_BSD_LIBRARY)
+	lte_lc_offline();
+	k_sleep(K_MSEC(100));
```

Comment:
Could use `k_yield()` here if the intention is to let other (same or higher priority) threads run